goon/browserassets/css/browserOutput.css

```diff
@@ -301,6 +307,7 @@ h1.alert, h2.alert			{color: #000000;}
 .rose					{color: #ff5050;}
 .sans					{font-family: 'Comic Sans MS', cursive, sans-serif;}
 .wingdings				{font-family: Wingdings, Webdings;}
+.ibm					{font-family: 'PxPlus IBM MDA';}
```

Comment:
@Terilia or @Ty-Omaha `font-size: '1.3em'` or some value > 1em here would probably help scale up the letters while not hardcoding the font size (it should grow and continue being the same size as normal characters if you increase font size in the chat window settings). `font-stye: italic` would italicize it along with the rest of the line, if the font supports italics. I can't test this out right now to make sure it'd look okay, otherwise I'd make the PR.